provisioning/atat_provisioning.yaml

```diff
@@ -406,263 +276,118 @@ components:
             - "IN_PROGRESS"
             - "COMPLETE"
             - "FAILED"
+    PortfolioPatch:
```

Comment:
Something about using the verb _patch_ in the name of the schema object strikes me as needing improvement, although patch can also be a noun.  Would `PortfolioAccess` be an improvement?  The operation that accepts this object is organized under the `access` tag.

Reply:
I think this is the comment you said I can disregard but to go ahead and answer - my idea was to make it forward compatible to whatever other attributes might need to go into the PATCH operation. Similarly, if we wanted to add an additional tag for a different functional area, we could do that too.